src/Lykke.Job.Messages/Sagas/BlockchainOperationsSaga.cs

```diff
@@ -56,6 +56,26 @@ public async Task Handle(SiriusDepositsDetector.Contract.Events.CashinCompletedE
             var walletId = string.IsNullOrWhiteSpace(evt.WalletId) ? (Guid?)null : Guid.Parse(evt.WalletId);
             await SendCashinEmailAsync(evt.OperationId, Guid.Parse(evt.ClientId), walletId, evt.Amount, evt.AssetId, commandSender);
         }
+        
+        //From InterestPayout
+        [UsedImplicitly]
+        public async Task Handle(InterestPayout.MessagingContract.PayoutCompletedEvent evt, ICommandSender commandSender)
+        {
+            var operationId = Guid.Parse(evt.OperationId);
+
+            var isHftWallet = evt.WalletId == evt.ClientId;
```

Comment:
It's opposite - hft wallet has id which does't match client id

Reply:
You can just rename it to isTradingWallet to avoid changing logic below

Reply:
`+`